pages/instructorAssessmentGroups/instructorAssessmentGroups.ejs

```diff
@@ -201,29 +201,27 @@
               </div>
               <form name="config-group-form" method="POST">
                 <div class="modal-body">
-                  <span class="badge badge-secondary">minimum size</span> <%= config_info.minimum %><br>
-                  <span class="badge badge-secondary">maximum size</span> <%= config_info.maximum %><br>
-                  <span class="badge badge-secondary">Student Permissions</span> <%= config_info.permission %><br>
-                  <br>
                   <div class="form-min">
                     <label for="formmin">Minimum group size</label>
-                    <input type="text" value=<%= config_info.defaultMin %> class="form-control" id="formmin" name="minsize">
+                    <input type="number" value=<%= config_info.defaultMin %> class="form-control" id="formmin" name="minsize">
                   </div>
+                  <br>
                   <div class="form-max">
                     <label for="formman">Maximum group size</label>
-                    <input type="text" value=<%= config_info.defaultMax %> class="form-control" id="formmax" name="maxsize">
+                    <input type="number" value=<%= config_info.defaultMax %> class="form-control" id="formmax" name="maxsize">
                   </div>
+                  <br>
                   <label for="formman">Student Permissions</label> <br>
                   <div class="form-check form-check-inline">
-                    <input class="form-check-input" type="checkbox" name="joincheck" value="true">
+                    <input class="form-check-input" type="checkbox" id = 'joincheck' name="joincheck" value="true">
```

Comment:
The new `id` attribute is not valid HTML. You have to use `"` quotes, not `'`. Also there should not be spaces around the `=`. However, I think we actually don't need this `id` at all (see below).

The right way to fix this is to set the `value` to be correct in the EJS template, based on the current database information.
```suggestion
                    <input class="form-check-input" type="checkbox" name="joincheck" value="<%- config_info.student_authz_join ? 'true' : 'false' %>">
```

Reply:
for the checkboxes, the value is actually not useful. I replaced the value with checked:
<%= config_info.student_authz_join ? "checked" : "" %>

Reply:
Oh yes, you are correct.